modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java

```diff
@@ -184,17 +211,9 @@ public void deleteResources()
         for (Service service : serviceManager.getAllServices()) {
             if (service.getQName().getLocalPart().equals("Info")) {
                 serviceManager.removeService(service.getId());
-            } else if (service.getQName().getLocalPart().equals("WeatherForecastService")) {
+            } else if (service.getQName().getLocalPart().equals("GeoIPService")) {
```

Comment:
The constant String literal should be the first part of the 'equals' check. 
Ex: else if("GeoIPService".equals(....)). 
Better to follow it as a practise, it avoids possible NPEs.